store_test.go

```diff
@@ -35,3 +61,8 @@ func (s *StoreSuite) TestLoadWithEmptyAggregateID() {
 
 	s.NoError(err)
 }
+
+func (s *StoreSuite) evtVersion(event *es.Event, version int64) *es.Event {
```

Comment:
We have to increment the versions to avoid optimistic locking... Perhaps we could think in a way to make events immutable 🤔